config.js

```diff
@@ -29,7 +29,7 @@ var config = {};
  * Default log level. Can be one of: 'DEBUG', 'INFO', 'WARN', 'ERROR', 'FATAL'
  * @type {string}
  */
-config.logLevel = 'INFO';
```

Comment:
Could you elaboratate on the changes in this file, please? Are they really needed?